Unity/PetEver/Assets/02.Scripts/Info/DogInformation.cs

```diff
@@ -22,13 +24,25 @@ void Start()
         controlDogInfoCG(false, null);
     }
 
+    private void parseDogInfo(string info)
+    {
+        //숑숑이,사과님의 10살 동생\\n비숑 ♀
+        dogInfoText = info.Split(',');
+    }
+
     private void controlDogInfoCG(bool showflag, Collider dog)
     {
         dogInfoCG = icanvas;
         if (showflag == true) {
             dogInfoCG.GetComponent<Canvas>().GetComponent<CanvasGroup>().alpha = 1;
             GameObject targetDog = GameObject.Find(dog.name);
             dogInfoCG.transform.GetChild(4).GetComponent<Image>().sprite = targetDog.transform.GetChild(0).GetChild(0).GetComponent<Image>().sprite;
+            
+            parseDogInfo(targetDog.transform.GetComponent<Text>().text);
+            dogInfoCG.transform.GetChild(1).GetComponent<TextMeshProUGUI>().text = dogInfoText[0];
```

Comment:
```suggestion
            GameObject dogName = GetChildWithName(dogInfoCG.gameObject, "PopupTitle");
            dogName.GetComponent<TextMeshProUGUI>().text = dogInfoText[0];
```

GetChild()문들을 이렇게 바꿔보는건 어떨까요? 

GetChildWithName은 별도로 만들어 주어야 합니다

```c#
GameObject GetChildWithName(GameObject obj, string childName)
{
        return obj.transform.Find(childName)?.gameObject;
}

```

Reply:
Applied your recommendation. Thanks.